tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -1916,40 +1918,63 @@ def graWorkGroup(self, kernel):
     ########################################
     # Blocked rows or columns
     # Do branch
-    wgmLabel = Label(label=self.labels.getNameInc("WGM"), comment="")
-    module.add(SCmpLeU32(src0=sgpr("WGM"), src1=1, comment="WGM <= 1 ?"))
-    module.add(SCBranchSCC1(labelName=wgmLabel.getLabelName(), comment="branch if WGM <= 1"))
+    wgmLabel         = Label(label=self.labels.getNameInc("WGM"), comment="")
+    wgmLabelNegative = Label(label=self.labels.getNameInc("WGMNegative"), comment="")
+    module.add(SCmpLeI32(src0=sgpr("WGM"), src1=1, comment="WGM <= 1 ?"))
+    module.add(SCBranchSCC1(labelName=wgmLabelNegative.getLabelName(), comment="branch if WGM <= 1"))
```

Comment:
To avoid jumping twice in WGM= 0 or 1 case, we can put negative version first.
Then, we can do like this.
if (WGM > 1)
  jump to positive code
Negative code
    here, we convert WGM to positive value
    if WGM<=1
        Jump to the end
Positive:
Positive code

Reply:
Fixed